src/3d/shaders/pointcloud.frag

```diff
@@ -16,14 +16,14 @@ uniform sampler1D u_colorRampTexture; //
 // Sets the color ramp value count, used to check the if not void
 uniform int u_colorRampCount;
 
-vec3 linearColorRamp()
+vec4 linearColorRamp()
```

Comment:
is this change still required?

Reply:
Returning RGBA values directly looks better to me